src/collection/dimensions.js

```diff
@@ -266,6 +266,20 @@ fn = elesfn = ({
         };
       }
 
+      function computePaddingValues( width, paddingObject ) {
```

Comment:
Yes I think it would be nice to have these options supported.  I can think of cases where I can use all of these options.